packages/vuetify/src/components/VBreadcrumbs/VBreadcrumbsItem.ts

```diff
@@ -38,6 +38,7 @@ export default mixins(Routable).extend({
         attrs: {
           ...data.attrs,
           'aria-current': this.isActive && this.isLink ? 'page' : undefined,
+          tabindex: this.disabled ? -1 : 0,
```

Comment:
We don't want to make non-disabled non-link items focusable either
```suggestion
          tabindex: this.disabled ? -1 : undefined,
```